src/menu/option-list.js

```diff
@@ -32,6 +32,18 @@ function OptionList(props: OptionListPropsT, ref: React.ElementRef<*>) {
     overrides.ListItem,
     StyledListItem,
   );
+  const getItem = item => {
+    return (
+      <>
+        {!item.href && getItemLabel(item)}
+        {item.href && (
+          <StyledListItemAnchor href={item.href} target="_blank">
```

Comment:
There will likely need to be an option to open in a new-tab/new-window/same-tab

Reply:
`StyledListItemAnchor ` has to be overrideable.

Reply:
yes, that would make perfect sense